### DISCUSSION TITLE
PSA: Unable to sign-in with a free account with no error provided!

### DISCUSSION THREAD

**Discussion by lkimloong** [SmoothWAN/SmoothWAN#86] [PSA]
Hi, wish to check if I have miss any tweaking on the smoothwan. Previously I have installed smoothwan 0.99.4 and I am able to log in to speedify thru the ui. I perform a fresh install to the latest version 0.99.8 and start install the speedify package. After speedify package installation I proceed to the speedify sign in and it keep bounce back to the sign in page after I key in my speedify credential. I try to downgrade the version it is still the same. Is there anything that I miss out?
Thanks

**Comment by TalalMash** [comment]
Hi, I just confirmed the same issue with a free account, unlimited (paid) accounts are not affected. Is yours paid?
Signing in to a paid account, logging out then in with the free account works.
I also noticed that they removed the X button to skip and use Speedify account-less. 

I've read that they have done something similarly few years ago to combat bots abusing the free service, I'd contact them and send the logs anyways, there is a generate logs button. (note that it also includes OpenWRT's network configuration)
The weird thing is that using `speedify_cli` works.

**Comment by TalalMash** [comment]
Same issue on Ubuntu, contact Speedify.

**Comment by lkimloong** [comment]
Wish to provide feedback that using a paid account is working. Previously I am logging in using a free account.

**Reply by TalalMash** [reply]
So I contacted Speedify and they simply said free accounts are not available on Linux, I'm very surprised to see that there is no indication or error reporting in the UI, and I can still connect via the CLI tool with the free 10GB data.

**Reply by lkimloong** [reply]
Can you provide guidance on how can we log in using the CLI tool?
Thanks

**Reply by TalalMash** [reply]
it's `/usr/share/speedify/speedify_cli login username pass` and the `connect` parameter in-place of login .
It actually seems to disconnect after few minutes.
